test/admin-hpa/run-local-tests.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/sh
+#set -x
+
+# Run the tests locally in parallel
+JOB_COUNT=6
```

Comment:
would be nice to make this an optional param with a default of 6, or is there a particular reason you chose this?

Reply:
6 seems a good choice for an 8 core system, and was chosen after a lot of tests runs from 5-10.  It works well.  If anyone has wildly more cores we can make it a param.